### DISCUSSION TITLE
Using ncc with Now

### DISCUSSION THREAD

**Discussion by sr229** [vercel/vercel#4087] [General] [answered]
Is there a way to use ncc with now? It seems rather a interesting concept but there's no docs that reference ncc being used for Now right now.

**Comment by styfle** [comment, marked as answer]
We used to use [`ncc`](https://github.com/zeit/ncc) for Node and Next.js deployments but we switched to [`node-file-trace`](https://github.com/zeit/node-file-trace) instead.

The difference is that it doesn't modify your source code so your errors produce the correct stack trace.

**Reply by sr229** [reply]
Ah, what's the difference between the two now?

**Reply by styfle** [reply]
The `node-file-trace` readme explains the difference.

> This package is used in @now/node and @now/next to determine exactly which files (including node_modules) are necessary for the application runtime.
> This is similar to @zeit/ncc except there is no bundling performed and therefore no reliance on webpack. This achieves the same tree-shaking benefits without moving any assets or binaries.

https://github.com/zeit/node-file-trace
